nodes/explorer/src/main.rs

```diff
@@ -11,14 +11,15 @@ use overwatch_rs::Services;
 use crate::api_backend::AxumBackend;
 use crate::config::{ApiArgs, Config};
 use nomos_log::Logger;
-use nomos_node::{LogArgs, Tx, Wire};
+use nomos_node::{DataAvailability, LogArgs, Tx, Wire};
 use nomos_storage::backends::sled::SledBackendSettings;
 use nomos_storage::{backends::sled::SledBackend, StorageService};
 
 #[derive(Services)]
 struct Explorer {
     log: ServiceHandle<Logger>,
     storage: ServiceHandle<StorageService<SledBackend<Wire>>>,
+    da: ServiceHandle<DataAvailability>,
```

Comment:
Why is there a da service here?

Reply:
Because in `da_blobs` fn, we need to connect to the `DataAvailability` service.

```rust
...

let relay = handle.relay::<DataAvailability>().connect().await?;

...
```

Reply:
This do not work as you think. The node explorer should not have this service.

Reply:
Ah, thanks, you are right. In this case, we should fetch the DA blobs from persistent storage. Currently, all of the DA blobs seem to be in the memory pool.

Reply:
We need to think about that indeed. Blobs can be retrieve from the node directly for now. But at some point we should derivate into something like storage.

Reply:
The blobs at the moment are stored in `BlobCache` which implements `DaBackend`. It has `get/add/remove` functions. It seems that this was created with an idea to be replaced with other storage. Maybe we could add a sled DaBackend there?

Reply:
I need to check. But in this case we could have a backend that keeps the memmory cache and the sled one. But definitely not do over this pr.